lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -273,6 +277,18 @@ def interpolate(self):
         interpolated_coord = np.interp(interpolated_time, self.time_idx, self.coordinate_idx)
         return self._with_coordinates(interpolated_time, interpolated_coord)
 
+    def _split(self, node):
+        """Split track
+
+        Parameters
+        ----------
+        node : int
+            Node index to split at
+        """
+        return self._with_coordinates(
+            self.time_idx[:node], self._localization[:node]
+        ), self._with_coordinates(self.time_idx[node:], self._localization[node:])
```

Comment:
nitpick: I don't really like this enforced formatting. Maybe clearer to just make two variables and then return both rather than all in one shot